spec/Candid.md

```diff
@@ -571,9 +583,9 @@ The types of these values are assumed to be known from context, so the syntax do
 <fieldval> ::= <nat> = <annval>
 
 <refval> ::=
-  | service <text>             (canister URI)
-  | func <text> . <name>       (canister URI and message name)
-  | principal <text>           (principal URI)
+  | principal <text>                      (principal URI)
```

Comment:
`principal` is now a primitive type, so we can move this to `<primval>`

Reply:
Done.